### DISCUSSION TITLE
PPOCRv4 rec and det English trained model download link

### DISCUSSION THREAD

**Discussion by neo133** [PaddlePaddle/PaddleOCR#12489] [Q&A] [answered]
Please provide PPOCRv4 recognition and detection trained model English language download link. Only the inference model is being downloaded by default. I want to fine-tune these weights over my own dataset.

**Comment by jinxiqinghuan** [comment, marked as answer]
You can download them from the following link:
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_ch/models_list.md

**Reply by Laubeee** [reply]
this link does not seem to privde PPOCRv4 models for english. They are still at v3..

**Reply by neo133** [reply]
Exactly what i asked..its not provided by them yet

**Reply by Laubeee** [reply]
I don't get why the issue is closed and the response is marked as answer if it's still not provided

**Reply by SWHL** [reply]
In this [docs](https://paddlepaddle.github.io/PaddleOCR/ppocr/model_list.html#21), it has provided.

Suggest reading more official documents

**Reply by Laubeee** [reply]
@SWHL unfortunately the official docs are quite a mess. Here are some obsevations:
- The link provided by @jinxiqinghuan and similarly the one you provided does contain en_PP-OCRv4_rec but not the det model.
- Any other models (I'd be interested in multilingual det + latin rec) are missing from all docs i have read
- the [english docs](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.8/doc/doc_en/models_list_en.md) do not mention v4 models at all

**Reply by SWHL** [reply]
I have received your feedback. Recently, PaddleOCR documents are transitioning to document sites, so it may appear a bit confusing. We will gradually clarify it in the later stage.

If you are interested, please feel free to help improve this document together.

**Reply by Laubeee** [reply]
I am happy to extend the english docs, but (see edit from my previous post, it seems I have wrongly remembered the v4-model from the old link) - there are still model files missing entirely.  (english detection, detection + recognition for other languages)
if they have already been trained, it would be great to have them uploaded, and if they are not, I'd be interested whether that is something that could happen in the near future

**Reply by SWHL** [reply]
Sorry to late slowly, I also found English model files missing entirely. 
You can directly use the corresponding model in Chinese, and the effect should also be good.
I'm not sure if there will be any in the future.

Welcome to contribute to the document.

**Reply by Laubeee** [reply]
Hi thanks for getting back to me about this
I just had a look at some more documents in chinese and according to [this page](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.8/doc/doc_ch/PP-OCRv4_introduction.md) the english and multilingual models have been trained and evaluated (see at the very end)... do you happen to know who is the correct person that would have access to these?

**Reply by SWHL** [reply]
I have access to this repo, welcome to your PR. After finishing, I will review it for you.
